ethergo/submitter/submitter.go

```
@@ -696,21 +765,35 @@
 	if err != nil {
 		return 0, fmt.Errorf("could not convert tx to call: %w", err)
 	}
-	// tmpdebug
-	fmt.Printf("Debug Calling EstimateGas")
 
-	gasEstimate, err = chainClient.EstimateGas(ctx, *call)
+	// bump gas limit from prior by X%
+	gasLimit_fromPrior := tx.Gas()
```

golangci-lint (1.62.2)

[warning] var-naming: don't use underscores in Go names; var gasLimit_fromPrior should be gasLimitFromPrior

(revive)